pyanaconda/ui/tui/spokes/installation_source.py

```diff
@@ -139,90 +141,77 @@ def refresh(self, args=None):
 
         threadMgr.wait(THREAD_PAYLOAD)
 
-        _methods = [_("CD/DVD"), _("local ISO file"), _("Network")]
+        self._container = ListColumnContainer(1, columns_width=78, spacing=1)
```

Comment:
I think this should be using constants as well. And possibly in the future we could even switch to terminal width autodetection, which we already use when formatting `--help` output.